CTLD.lua

```diff
@@ -678,21 +678,21 @@ ctld.i18n["es"]["1 FOB Crate oboard (%1 kg)\n"] = "1 caja FOB a bordo (%1 kg)\n"
-ctld.i18n["es"]["Hovering above %1 crate. \n\nHold hover for %2 seconds! \n\nIf the countdown stops you're too far away!"] = "Stationaire au-dessus de la caisse %1. \n\nMaintenez le stationaire pendant %2 secondes ! \n\nSi le compte à rebours s'arrête, vous êtes trop loin !"
-ctld.i18n["es"]["Loaded %1 crate!"] = "Caisse %1 chargée !"
-ctld.i18n["es"]["Too low to hook %1 crate.\n\nHold hover for %2 seconds"] = "Trop bas pour accrocher la caisse %1.\n\nMaintenez le stationaire pendant %2 secondes"
-ctld.i18n["es"]["Too high to hook %1 crate.\n\nHold hover for %2 seconds"] = "Trop haut pour accrocher la caisse %1.\n\nMaintenez le stationaire pendant %2 secondes"
-ctld.i18n["es"]["You must land before you can load a crate!"] = "Vous devez atterrir avant de pouvoir charger une caisse !"
-ctld.i18n["es"]["No Crates within 50m to load!"] = "Aucune caisse à moins de 50 m pour charger !"
-ctld.i18n["es"]["Maximum number of crates are on board!"] = "Nombre maximal de caisses à bord !"
-ctld.i18n["es"]["%1\n%2 crate - kg %3 - %4 m - %5 o'clock"] = "%1\n%2 caisse - kg %3 - %4 m - %5 heures"
-ctld.i18n["es"]["FOB Crate - %1 m - %2 o'clock\n"] = "Caisse FOB - %1 m - %2 heures\n"
-ctld.i18n["es"]["No Nearby Crates"] = "Aucune caisse à proximité"
-ctld.i18n["es"]["Nearby Crates:\n%1"] = "Caisses à proximité :\n%1"
-ctld.i18n["es"]["Nearby FOB Crates (Not Slingloadable):\n%1"] = "Caisses FOB à proximité (non chargeables par élingue) :\n%1"
-ctld.i18n["es"]["FOB Positions:"] = "Positions FOB :"
+ctld.i18n["es"]["Hovering above %1 crate. \n\nHold hover for %2 seconds! \n\nIf the countdown stops you're too far away!"] = "Flotando sobre %1 caja. \n\n¡Manténte flotando sobre ella durante %2 segundos! \n\n¡Si la cuenta regresiva se detiene, estás demasiado lejos!"
+ctld.i18n["es"]["Loaded %1 crate!"] = "Caja %1 cargada !"
+ctld.i18n["es"]["Too low to hook %1 crate.\n\nHold hover for %2 seconds"] = "Demasiado bajo para enganchar la caja %1.\n\nManténte flotando sobre ella durante %2 segundos"
+ctld.i18n["es"]["Too high to hook %1 crate.\n\nHold hover for %2 seconds"] = "Demasiado alto para enganchar la caja %1.\n\nManténte flotando sobre ella durante %2 segundos"
+ctld.i18n["es"]["You must land before you can load a crate!"] = "¡Debes aterrizar antes de poder cargar una caja!"
+ctld.i18n["es"]["No Crates within 50m to load!"] = "¡No hay cajas para cargar a 50 m a la redonda!"
+ctld.i18n["es"]["Maximum number of crates are on board!"] = "¡El número máximo de cajas está a bordo!"
+ctld.i18n["es"]["%1\n%2 crate - kg %3 - %4 m - %5 o'clock"] = "%1\n%2 caja - kg %3 - %4 m - %5 horas"
+ctld.i18n["es"]["FOB Crate - %1 m - %2 o'clock\n"] = "Caja FOB - %1 m - %2 horas\n"
+ctld.i18n["es"]["No Nearby Crates"] = "Ninguna caja de proximidad"
+ctld.i18n["es"]["Nearby Crates:\n%1"] = "Cajas cercanas:\n%1"
+ctld.i18n["es"]["Nearby FOB Crates (Not Slingloadable):\n%1"] = "Cajas FOB cercanas (no transportable por cable):\n%1"
+ctld.i18n["es"]["FOB Positions:"] = “Posiciones FOB:”
```

Comment:
Attention à prendre les bons guillemets !

```suggestion
ctld.i18n["es"]["FOB Positions:"] = "Posiciones FOB:"
```

Reply:
Désolé pour ces erreurs. Je ne comprends pas comment j'ai pu les louper, car j'ai testé cette version avant e l'envoyer. Logiquement j'aurais dû avoir une erreur de syntaxe de LUA. 
Je serai plus vigilant à l'avenir.